runtime/doc/treesitter.txt

```diff
@@ -1055,6 +1055,10 @@ edit({lang})                                     *vim.treesitter.query.edit()*
     Parameters: ~
       • {lang}  (`string?`) language to open the query editor for. If omitted,
                 inferred from the current buffer's filetype.
+      • {opts}  (`table?`) Optional keyword arguments:
```

Comment:
For newly added `opts` parameters, I always prefer creating a Lua type for it to get that sweet completion support.